nixos/modules/virtualisation/google-compute-image.nix

```diff
@@ -44,10 +44,20 @@ in
         GZIP compression level of the resulting disk image (1-9).
       '';
     };
+    virtualisation.googleComputeImage.efi =
+      mkEnableOption (lib.mdDoc "Whether to use EFI booting") // {
+        default = false;
```

Comment:
This is default already: https://github.com/nixos/nixpkgs/blob/master/lib/options.nix#L101-L101

Reply:
Thank you. Updated